pg_search/src/postgres/merge.rs

```diff
@@ -309,8 +328,13 @@ unsafe extern "C-unwind" fn background_merge(arg: pg_sys::Datum) {
             return;
         }
         let index = index.unwrap();
+        let sentinel_buffer = MetaPage::open(&index)
+            .bgmerger()
+            .try_starting(args.blockno());
+        if sentinel_buffer.is_none() {
+            return;
```

Comment:
It's failing silently here. Shouldn't we at least log something here? It'd be better if we could collect a metric here.